src/annotator/highlighter.js

```diff
@@ -78,6 +233,10 @@ export function removeHighlights(highlights) {
       const children = Array.from(h.childNodes);
       replaceWith(h, children);
     }
+
+    if (h.svgHighlight) {
```

Comment:
Is there any case where svgHighlight prop would be missing and would that be a problem?

Reply:
This property won't be set for highlights on HTML content (ie. not a PDF), in which case it will just evaluate to `undefined` here and this branch will be skipped.